test/configuration_test.rb

```diff
@@ -187,25 +187,47 @@ def hash
     end
   end
 
-  describe '#default_format' do
```

Comment:
@theocodes We need both the tests for `#default_format` and `#default_request_format`. The first is still there, and we want to make sure that it still works. :wink: